README.md

```diff
@@ -369,4 +370,255 @@ You may find it helpful to see the Kubernetes resources visually which can be ac
+
+
+# Docker for Desktop - Windows
+
+### 1. Install Docker for Desktop
+
+Check recommended version [here](https://github.com/Alfresco/alfresco-dbp-deployment/blob/master/README-prerequisite.md#docker-desktop).
+
+### 2. Enable Kubernetes
+
+In the 'Kubernetes' tab of the Docker settings, click the 'Enable Kubernetes' checkbox.
+
+Run Command Prompt as an administrator.
+
+Enter the following commands to delete the storageClass hostpath and set up the hostpath provisioner: 
+
+```bash
+kubectl delete storageclass hostpath
+kubectl create -f https://raw.githubusercontent.com/MaZderMind/hostpath-provisioner/master/manifests/rbac.yaml
+kubectl create -f https://raw.githubusercontent.com/MaZderMind/hostpath-provisioner/master/manifests/deployment.yaml
+kubectl create -f https://raw.githubusercontent.com/MaZderMind/hostpath-provisioner/master/manifests/storageclass.yaml
+```
+
+### 3. Increase Memory and CPUs
+
+In the Advanced tab of the Docker preferences, set 'CPUs' to 4.
+
+While Alfresco Digital Business Platform installs and runs with only 8 GiB allocated to Docker, 
+for better performance we recommend that 'Memory' value be set slightly higher, to at least 10 - 12 GiB
+(depending on the size of RAM in your workstation). 
+
+### 4. Change/Verify Context
+
+If you have previously deployed the DBP to AWS or minikube you will need to change/verify that the `docker-for-desktop` context is being used.
+
+```bash
+kubectl config current-context                 # Display the current context
+kubectl config use-context docker-for-desktop  # Set the default context if needed
+```
+
+### 5. Restart Docker  
+
+Docker can be faulty on its first start. So, it is always safer to restart it before proceeding. Right click on the Docker icon in the system tray, then left click "restart...". 
+
+### 6. Install Helm
+
+Install chocolatey using command prompt (with admin rights). 
+
+```bash
+@"%SystemRoot%\System32\WindowsPowerShell\v1.0\powershell.exe" -NoProfile -InputFormat None -ExecutionPolicy Bypass -Command "iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))" && SET "PATH=%PATH%;%ALLUSERSPROFILE%\chocolatey\bin"
+```
+
+Install helm
+
+```bash
+choco install kubernetes-helm
+```
+
+Initialize Tiller (Server Component)
+
+```bash
+helm init
+```
+
+### 7. Create your namespace
+
+```bash
+kubectl create namespace $DESIREDNAMESPACE
+```
+
+### 8. Pull secrets
+
+Log in to quay.io
+```bash
+docker login quay.io
+```
+Give username and password if prompted.
+
+In windows explorer, go to the current location of your command line.
```

Comment:
Instead of the next few commands I think you can simplify that to the following command that can be run from powerShell :
```bash
echo "apiVersion: v1
kind: Secret
metadata:
  name: quay-registry-secret
type: kubernetes.io/dockerconfigjson
data:
  .dockerconfigjson: $QUAY_SECRET" > secrets.yaml
```

Reply:
oh that seems cool. Thanks Elena. But we should still do the steps of pasting the string right? Because secrets.yaml still needs the token
![image](https://user-images.githubusercontent.com/42536841/53156662-ca402700-35b7-11e9-9ed4-9129f0d11374.png)
![image](https://user-images.githubusercontent.com/42536841/53156702-dd52f700-35b7-11e9-959c-8dd7680ed425.png)

Reply:
Ignore this. Chris explained to me what you mean and it is done now :)